mycroft/client/speech/hotword_factory.py

```diff
@@ -191,7 +192,17 @@ def __init__(self, key_phrase="hey mycroft", config=None, lang="en-us"):
         from precise_runner import (
             PreciseRunner, PreciseEngine, ReadWriteStream
         )
-        local_conf = LocalConf(USER_CONFIG)
+
+        old_path = join(expanduser('~'), './mycroft/mycroft.conf')
+        if isfile(old_path):
+            local_conf = old_path
+        else:
+            for dir in BaseDirectory.load_config_paths('mycroft'):
+                local_conf = LocalConf(join(dir, 'mycroft.conf'))
+                # If the current config contains the precise key use it,
+                # otherwise continue to the next file
+                if local_conf.get('precise', None) is not None:
+                    break
```

Comment:
This may not work as intended if the precise conf is made in the xdg location but the old .mycroft location still exists. Should be quite a rare case though...

Also does the `BaseDirectory.load_config_paths()` function also include locations not writable by the user? in that case the `local_conf.store()` below may fail and throw an exception.

Reply:
I guess we should give preference to the new XDG location, and only use the old location if no `precise` key is found yet?

`load_config_paths()` can indeed return unwriteable locations, I now made sure to always save to a writeable location but read from unwriteable ones too.

Reply:
Yeah make preference to the new XDG based location sounds like the best option

Reply:
Cool, that's what I did now.

That's all conversations resolved then :tada: